backend/routes/usernameRoute.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+
+router.post('/check-username',  async (req, res) => {
```

Comment:
The handler function and its logic that you have currently living in the endpoints should be extracted into functions and then just import it in the routes file accordingly. The functions should live in a folder called handlers. So in your case you'd import a method that handles the changeUsername logic from handlers/changeUsername.js or something along those lines.